resources/assets/coffee/react/contest-page.coffee

```diff
@@ -0,0 +1,28 @@
+###*
+*    Copyright 2015 ppy Pty. Ltd.
+*
+*    This file is part of osu!web. osu!web is distributed with the hope of
+*    attracting more community contributions to the core ecosystem of osu!.
+*
+*    osu!web is free software: you can redistribute it and/or modify
+*    it under the terms of the Affero GNU General Public License version 3
+*    as published by the Free Software Foundation.
+*
+*    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+*    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+*    See the GNU Affero General Public License for more details.
+*
+*    You should have received a copy of the GNU Affero General Public License
+*    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+*
+###
+propsFunction = ->
+  tracks: osu.parseJson('json-tracks')
+  options:
+    hideGenre: true
+    hideBPM: true
+    hideDL: true
+    showVote: true
+    smaller: true
+
+reactTurbolinks.register 'tracklist', Tracklist, propsFunction
```

Comment:
Pretty sure registering two components with same class name [doesn't work](/ppy/osu-web/blob/master/resources/assets/coffee/_classes/react-turbolinks.coffee#L39) unless turbolinks is disabled.